plasmapy/utils/tests/collections_helpers_tests.py

```diff
@@ -0,0 +1,100 @@
+import unittest
+from collections import defaultdict
+from plasmapy.utils import compare_default_dicts
+
+
+class TestDictCompare(unittest.TestCase):
```

Comment:
I was a bit confused as to why you'd use `unittest` here, but sure, if you want to write them that way :) Pytest handles unittest-style tests, so it's fine.